tower/src/balance/p2c/make.rs

```diff
@@ -12,15 +12,22 @@ use std::{
 };
 use tower_service::Service;
 
-/// Makes `Balancer`s given an inner service that makes `Discover`s.
+/// Construct load balancers over dynamic service sets produced by a wrapped "inner" service.
```

Comment:
nit: I think this should be
```suggestion
/// Constructs load balancers over dynamic service sets produced by a wrapped "inner" service.
```